### PR TITLE
Changed formatters to use configuration #444

### PR DESCRIPTION
**Pull request by joachimmetz**
Changed formatters to use configuration #444

### CODE REVIEW

**Comment by joachimmetz** [comment]
@rgayon PTAL

File: data/formatters/generic.yaml [hunk 1]
```diff
@@ -150,6 +168,113 @@ short_source: 'HIST'
...
+---
+type: 'conditional'
+data_type: 'skydrive:log:line'
+message:
+- '[{module}'
+- '{source_code}'
+- '{log_level}]'
+- '{detail}'
+short_message:
+- '{detail}'
```
Comment (rgayon) [inline data/formatters/generic.yaml:189]: Just noticed the [] got dropped from the previous formatter, is this WAI?

Comment (rgayon) [reply data/formatters/generic.yaml:189]: Same in other short_messages

Comment (joachimmetz) [reply data/formatters/generic.yaml:189]: Not sure:
* module and log_level contain `[]`
* the other `[]` are for the lists

Could you double check/confirm

```
class SkyDriveLogFormatter(interface.ConditionalEventFormatter):	
  """Formatter for a SkyDrive log file event."""	

  DATA_TYPE = 'skydrive:log:line'	

  FORMAT_STRING_PIECES = [	
      '[{module}',	
      '{source_code}',	
      '{log_level}]',	
      '{detail}']	

  FORMAT_STRING_SHORT_PIECES = ['{detail}']	

  SOURCE_LONG = 'SkyDrive Log File'	
  SOURCE_SHORT = 'LOG'
```

Comment (rgayon) [reply data/formatters/generic.yaml:189]: oh right I see now, was confused

Comment (joachimmetz) [reply data/formatters/generic.yaml:189]: noworries, thx for confirming

**Pull request merged**